test/playerTest.js

```diff
@@ -0,0 +1,75 @@
+var chai = require('chai'),
+expect = chai.expect,
+path = require('path');
+
+
+// Tell chai that we'll be using the "should" style assertions.
+chai.should();
+
+// Import the Player & Phaser class.
+var Character = require(path.join(__dirname, '../src/', 'player'));
+
+describe('Player', () => {
+  describe('#name', () => {
+    let player;
+    let playerName = 'Bob'
+
+    beforeEach(() => {
+      // Create a new Player object before every test.
+      player = new Character(playerName);
+    });
+
+    it('return player name', () => {
+      // This will fail if "player.name" does
+      // not equal to Bob.
+      player.name = playerName;
+      player.name.should.equal(playerName);
+    });
+
+    it('player name can be changed', () => {
+      // Assert that the width can be changed.
+      let playerName = 'Tom'
+      player.name = playerName;
+      player.name.should.equal(playerName);
+    });
+
+    it('only accepts string values of names', () => {
```

Comment:
No need for testing that. We need to test game functionality overall not the library and small bugs